CodeStarterCamp_Week3/CoffeeShop.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  CoffeeShop.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by 김예림 on 2023/06/01.
+//
+
+import Foundation
+
+class CoffeeShop {
+    var sales: Int
+    var menu: [(coffee: String, price: Int)]
+    var pickUpTable: [Coffee]
+    
```

Comment:
typealias로 menu를 지정해 주는 것도 좋을 것 같아요 !

```suggestion

typealias menu = (coffee: String, price: Int)

struct CoffeeShop {
    var sales: Int
    var menu: [menu]
    var pickUpTable: [Coffee]
    
```